test/test_spectral_ops.py

```diff
@@ -566,11 +467,7 @@ def fn(t: torch.Tensor, s: Optional[List[int]], dim: List[int] = (-2, -1), norm:
 
                 torch_fns = (torch_fn, torch.jit.script(fn))
 
-                if torch.version.hip is not None:
-                    valid_input_default, s, _, norm = self._generate_valid_rocfft_input(
-                        input, torch_fn, s, None, norm)
-                else:
-                    valid_input_default = input
+                valid_input_default = input
```

Comment:
Can we just use input throughout the test now and remove both `valid_input_default` and `valid_input_explicit` which are the same thing?

Reply:
Done